subcommands/waves/status.go

```diff
@@ -134,6 +134,23 @@ func doShowWaveStatus(cmd *cobra.Command, args []string) {
 			}
 		}
 	}
+
+	if len(status.RolloutHistory) > 0 {
+		fmt.Println("\nRollout history:")
+		t = subcommands.Tabby(0, "ROLLOUT TO", "ROLLOUT BY", "ROLLOUT AT")
+		var action string
+		for _, rollout := range status.RolloutHistory {
+			if rollout.IsFullGroup {
+				action = fmt.Sprintf("all devices in group %s", rollout.Group)
+			} else if rollout.IsFactoryWide {
+				action = fmt.Sprintf("%d devices in factory", rollout.DeviceNumber)
+			} else {
+				action = fmt.Sprintf("%d devices in group %s", rollout.DeviceNumber, rollout.Group)
+			}
+			t.AddLine(action, rollout.RolloutBy, rollout.RolloutAt)
+		}
+		t.Print()
```

Comment:
I think it is worth showing a history line like `all devices in factory` in case if a wave was completed.
(I'd need to make a small change to the API to return a `finished-by` field for that).
What do you think?

Reply:
how does that wind up rendering with Tabby?

Reply:
one thing you don't see is what wave or update the rollout was for. Is there anyway to provide that context?

Reply:
This line is printed at the top (see example above):
```
Wave 'test-dspl-prod-wave-active' for tag 'master' version 21 is active
```